src/api/realtime/realTimeModule.ts

```diff
@@ -1,13 +1,22 @@
 import { ReflectiveInjector } from "injection-js";
+import { IResource } from "jexia-sdk-js/api/core/resource";
```

Comment:
relative path?

Reply:
yeah, IDE put this because of `path` in `typescript.json`. Wishing to get rid of this :)

Reply:
I knew it :laughing: